### PR TITLE
fix: minor ui bugs

### LINKED ISSUE #4561: Werner UI feedback
Making issue of werner's comment on closed PR

              1. Increasing fee on STX transaction, this looks to big/not centered
image

2. Send SIP010 token, "edit nonce" overlapping and not clickable:
It looks like the footer 'available balance .... X.XXXXXXXXXXXX' here isn't on top, "edit nonce" should go behind as it does in the "send STX page". 
image

3. Title should remain on top, so the user can always click "x" to close?
image
4. To and address not a alligned
image

5. Should these lines be here? (see this when using app.alexlab.co and creating a swap to sign)
image

Probably unrelated to this PR. 
6. With ledger trying a swap I see this... API issues? Transaction submitted but not broadcasted... trying a transfer instead the transaction fails outright with "transaction rejected". 
image

Sending STX instead... looks like this is happening with 6.16.1 too on testnet... on mainnet it is fine. 
image

_Originally posted by @314159265359879 in https://github.com/leather-wallet/extension/issues/4490#issuecomment-1822404811_

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out this version of Leather — [download extension builds](https://github.com/leather-wallet/extension/actions/runs/6975852717).

Fixes multiplier buttons:
Screenshot 2023-11-23 at 2 57 36 PM

Fixes missing background on footer:
Screenshot 2023-11-23 at 3 06 08 PM

Removes underline:
Screenshot 2023-11-23 at 7 24 41 PM

### CODE REVIEW

File: src/app/components/info-card/info-card.tsx [hunk 1]
```diff
@@ -126,7 +126,7 @@ export function InfoCardFooter({ children }: InfoCardFooterProps) {
       alignItems="center"
       bg={whenPageMode({
         full: '',
-        popup: '#fff',
+        popup: 'accent.background-primary',
```
Comment (kyranjamie) [inline src/app/components/info-card/info-card.tsx:129]: You're certain this works? Wondered whether the static analysis of panda works when wrapping in the whenPageMode

File: theme/typography.ts [hunk 2]
```diff
@@ -90,12 +90,17 @@ export const textStyles = defineTextStyles({
...
 
   'mono.01': {
     description: 'mono.01',
-    value: { fontFamily: firaCode, fontSize: '1.0625rem', lineHeight: '1.75rem' },
+    value: {
+      fontFamily: firaCode,
+      fontSize: '1rem',
+      lineHeight: '1.5rem',
+      letterSpacing: '.08rem',
+    },
```
Comment (kyranjamie) [inline theme/typography.ts:103]: @mica000 @fabric-8 are these changes consistent with design system?

**Review by kyranjamie — APPROVED** [review]

File: src/app/pages/rpc-send-transfer/components/send-transfer-footer.tsx [hunk 3]
```diff
@@ -11,10 +11,7 @@ export function SendTransferFooter({ children }: HasChildren) {
         full: 'unset',
         popup: 'accent.background-primary',
       })}
-      // TODO #4476 check this border colour it was EFEFF2
-      // - can we pass it with shorthand?
-      borderTop="1px solid"
-      borderColor="border-default"
+      borderTop="default"
```
Comment (pete-watters) [inline src/app/pages/rpc-send-transfer/components/send-transfer-footer.tsx:14]: 👏

**Review by pete-watters — APPROVED** [review]
Nice work!

File: src/app/components/info-card/info-card.tsx (hunk 1 above)
Comment (fbwoolf) [reply src/app/components/info-card/info-card.tsx:129]: Yes, I did check it.

File: theme/typography.ts (hunk 2 above)
Comment (fbwoolf) [reply theme/typography.ts:103]: I did check this in the designs, that is why I made the change here.

Comment (fbwoolf) [reply theme/typography.ts:103]: Image of change for ref, I did check the review page designs so let me know if not accurate...
 
Screenshot 2023-11-24 at 12 23 16 PM

File: src/app/components/info-card/info-card.tsx (hunk 1 above)
Comment (fbwoolf) [reply src/app/components/info-card/info-card.tsx:129]: Confirmed, before under the button was actually rendering with no bg color...

Screenshot 2023-11-24 at 11 52 39 AM

Comment (fbwoolf) [reply src/app/components/info-card/info-card.tsx:129]: Better image to show footer bg is now solid...

Screenshot 2023-11-24 at 12 39 09 PM

**Pull request merged**